openslides_backend/action/actions/meeting_user/mixin.py

```diff
@@ -10,6 +11,77 @@
 
 
 class MeetingUserMixin(Action):
+    standard_fields = [
+        "comment",
+        "number",
+        "structure_level",
+        "vote_weight",
+        "personal_note_ids",
+        "speaker_ids",
+        "supported_motion_ids",
+        "submitted_motion_ids",
+        "assignment_candidate_ids",
+        "vote_delegated_vote_ids",
+        "vote_delegated_to_id",
+        "vote_delegations_from_ids",
+        "chat_message_ids",
+    ]
+
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
+        """ standard_fields have to be checked, but also it is not permitted, create an empty meeting_value instance"""
+        if any(fname in self.standard_fields for fname in instance.keys()) or not any(fname in ["about_me", "group_ids"] for fname in instance):
+            return super().check_permissions(instance)
+
+        def get_user_and_meeting_id() -> Tuple[int]:
+            fields = ["user_id", "meeting_id"]
+            if any(fname not in instance for fname in fields):
+                mu = self.datastore.get(
+                    fqid_from_collection_and_id("meeting_user", instance["id"]),
+                    ["user_id", "meeting_id"],
+                    lock_result=False,
+                )
+                return (mu[fname] for fname in fields)
+            return (instance[fname] for fname in fields)
+
+        def get_request_user_data() -> Dict[str, Any]:
+            return self.datastore.get(
+                fqid_from_collection_and_id("user", self.user_id),
+                ["organization_management_level", "committee_management_ids"],
+                lock_result=False,
+            )
+
+        def get_committee_id() -> int:
+            return self.datastore.get(
+                fqid_from_collection_and_id("meeting", meeting_id),
+                ["committee_id"],
+                lock_result=False,
+            )["committee_id"]
+
+        def raise_own_exception() -> bool:
+            try:
+                super().check_permissions(instance)
+                return False
+            except:
+                return True
+
+        user_id, meeting_id = get_user_and_meeting_id()
+        if "about_me" in instance:
+            if self.user_id != user_id:
+                if raise_own_exception():
+                    raise PermissionDenied(f"The user needs Permission user.can_manage in meeting {meeting_id} to set 'about me', if it is not his own")
```

Comment:
Do you really need the extra error message or can you just use the `MissingPermission` exception?

Reply:
There is no permission for user.can_change_self to express one of the missing permissions.